validator/client/service.go

```diff
@@ -250,3 +253,26 @@ func ConstructDialOptions(
 
 	return dialOpts
 }
+
+// fetch Graffiti from a file or flag.
+func fetchGraffiti(flag string) [][]byte {
+	if flag == "" {
+		// No graffiti.
+		return make([][]byte, 1)
```

Comment:
This mirrors the existing code in that it goes from being a []byte variable of nil to an array of []byte with a nil entry.  It would be possible to just pass nil here, but that would then require logic to handle the array / no array cases.